noir-projects/aztec-nr/aztec/src/discovery/private_logs.nr

```diff
@@ -50,7 +48,7 @@ pub unconstrained fn fetch_and_process_private_tagged_logs<Env>(
 /// search for the note's completion public log.
 pub unconstrained fn do_process_log<Env>(
     contract_address: AztecAddress,
-    log: BoundedVec<Field, PRIVATE_LOG_CIPHERTEXT_LEN>,
+    log: BoundedVec<Field, PRIVATE_LOG_SIZE_IN_FIELDS>,
```

Comment:
This changed is caused by me re-introducing the tag here because I needed that info to figure out which logs are duplicates in `PrivateEventDataProvider`.